src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -2807,6 +2809,13 @@ ObjectList* SmartScript::GetTargets(SmartScriptHolder const& e, Unit* invoker /*
                 }
             }
         }
+        case SMART_TARGET_SUMMONER:
+        {
+            if (me && me->ToTempSummon())
+                if (Unit* summoner = me->ToTempSummon()->GetSummoner())
+                    l->push_back(me->ToTempSummon()->GetSummoner());
```

Comment:
Right, no idea how I managed to forget about that after putting it there. Thanks :P